app/views/ExposureHistory/History.tsx

```diff
@@ -74,11 +75,7 @@ const History = ({ exposureHistory }: HistoryProps): JSX.Element => {
             </TouchableOpacity>
           </View>
           <View style={styles.headerRow}>
-            {!isGPS ? (
-              <Typography style={styles.subHeaderText}>
-                {lastDaysText}
-              </Typography>
-            ) : null}
+            {isBT ? <DateInfoHeader /> : null}
```

Comment:
is there a way we can do this without this conditional?  Does the GPS side have a similar feature requirement?

Reply:
if we do the call in the Exposure history context, and have the component not render if there isnt the data, then we should be able to remove the conditional altogether.  If the GPS side eventually wants to use it, they simply need to provide a posix of the last updated at time.

Reply:
I think that calling something regardless could cause a warning like the one we were seeing on the middleware for the intersect service since this module won't be bundled on the GPS build. We could use the conditional on the `existence` of the module perhaps?

Reply:
I think we can make it conditionally render now that we have a value depending on an actual module response only available on the BT flavor. Good catch!